class.jetpack-gutenberg.php

```diff
@@ -23,6 +23,12 @@ function jetpack_register_block( $slug, $args = array() ) {
 		_doing_it_wrong( 'jetpack_register_block', 'Prefix the block with jetpack/ ', '7.1.0' );
 		$slug = 'jetpack/' . $slug;
 	}
+
+	// Checking whether block is registered to ensure it isn't registered twice.
+	if ( Jetpack_Gutenberg::is_registered( $slug ) ) {
+		return false;
+	}
```

Comment:
The thing is, this sort of check is kinda what `register_block_type()` does, and what results in the warning that we're trying to fix here (#11590) :sweat_smile: So in a way, we're just cutting in line before that check to silence that warning, when the underlying issue is that some consumer registers a block twice (which it shouldn't). So _ideally_, we'd find out what consumer that is and change it accordingly :grimacing:

Reply:
> ideally, we'd find out what consumer that is and change it accordingly

There is a use-case here for example:
https://github.com/Automattic/jetpack/issues/11506#issuecomment-470330131

Reply:
Seems like both are valid concerns but given the time constraints, this seems like a valid way to go for now?

Reply:
Yep: https://github.com/Automattic/jetpack/pull/11618#pullrequestreview-218003272